pytorch_lightning/core/hooks.py

```diff
@@ -669,6 +669,9 @@ def transfer_batch_to_device(self, batch: Any, device: Optional[torch.device] =
         Note:
             This hook should only transfer the data and not modify it, nor should it move the data to
             any other device than the one passed in as argument (unless you know what you are doing).
+            To check the current state of execution of this hook you can use
+            ``self.trainer.training/testing/validating/predicting`` so that you can
+            add different logic as per your requirement.
```

Comment:
This sentence could be added to every hook we have in Lightning so I'm wondering how relevant it is here.

Reply:
not every. All other hooks are different for train/val/test(for eg on_train_.., on_test...) but this one is a common one for all so there is a possibility that the user needs to have different transformations in train/val/test, that's why added it here for reference.